### ISSUE TITLE
Tasklist logs are full of warnings in core setup

### ISSUE THREAD

**Issue by ingorichtsmeier** [camunda/camunda-platform#17]
The logs of tasklist are full fo warnings:

```
2022-05-18 11:36:15.327  WARN 1 --- [       Thread-2] o.e.c.RestClient                         : request [POST http://elasticsearch:9200/zeebe-record-variable/_search?routing=1&typed_keys=true&max_concurrent_shard_requests=5&request_cache=false&search_type=query_then_fetch&batched_reduce_size=512] returned 1 warnings: [299 Elasticsearch-7.17.0-bee86328705acaa9a6daede7140defd4d9ec56bd "Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.17/security-minimal-setup.html to enable security."]
```

As security is no concern by intend, the warning should be suppressed.

They are printed several times every few seconds (I assume for each request).

**Comment by ingorichtsmeier** [comment]
here is a hint how to suppress the messages: https://stackoverflow.com/a/70421802

**Comment by ingorichtsmeier** [comment]
Would be solved with this pull request: #18
